apps/products/models.py

```diff
@@ -0,0 +1,250 @@
+    type = models.CharField(max_length=10, choices=TYPE_CHOICES, blank=True, null=True)
+    code = models.CharField(max_length=50)
+    hsn_or_sac_code = models.CharField(max_length=50)
+    hsn_description = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = productgstclassificationstable
+
+    def __str__(self):
+        return f"{self.gst_classification_id} {self.code}"
+
+
+class ProductSalesGl(models.Model):
+    sales_gl_id = models.AutoField(primary_key=True)
+    ledger_group_id = models.ForeignKey(LedgerGroups, on_delete=models.CASCADE, null=True, default=None, db_column = 'ledger_group_id')
+    name = models.CharField(max_length=255)
+    sales_accounts = models.CharField(max_length=255)
+    code = models.CharField(max_length=50)
+    is_subledger = models.BooleanField(default=False)
+    inactive = models.BooleanField(default=False)
+    type = models.CharField(max_length=255)
+    account_no = EncryptedTextField(max_length=255)
+    rtgs_ifsc_code = models.CharField(max_length=255)
+    classification = models.CharField(max_length=255)
+    is_loan_account = models.BooleanField(default=False)
+    tds_applicable = models.BooleanField(default=False)
+    address = models.CharField(max_length=255)
+    pan = models.CharField(max_length=50)
+    employee = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'product_sales_gl'
+
+    def __str__(self):
+        return f"{self.sales_gl_id} {self.name}"
+
+class ProductPurchaseGl(models.Model):
+    purchase_gl_id = models.AutoField(primary_key=True)
+    ledger_group_id = models.ForeignKey(LedgerGroups, on_delete=models.CASCADE, null=True, default=None, db_column = 'ledger_group_id')
+    name = models.CharField(max_length=255)
+    purchase_accounts = models.CharField(max_length=255)
+    code = models.CharField(max_length=50)
+    is_subledger = models.BooleanField(default=False)
+    inactive = models.BooleanField(default=False)
+    type = models.CharField(max_length=255)
+    account_no = EncryptedTextField(max_length=255)
+    rtgs_ifsc_code = models.CharField(max_length=255)
+    classification = models.CharField(max_length=255)
+    is_loan_account = models.BooleanField(default=False)
+    tds_applicable = models.BooleanField(default=False)
+    address = models.CharField(max_length=255)
+    pan = models.CharField(max_length=50)
+    employee = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'product_purchase_gl'
+
+    def __str__(self):
+        return f"{self.purchase_gl_id} {self.name}"
+
+
+def products_picture(instance, filename):
```

Comment:
why this method is outside the class? if it is common method and can be reuesed then move to utils_methods.py file

Reply:
Actually from picture column we are calling that method Thulasi
    picture = models.ImageField(max_length=255, default=None, null=True, upload_to=products_picture)
and we tried putting it in utils_methods but everyone having different requirements so we left like that only .